src/components/stories/Stories.jsx

```diff
@@ -1,20 +1,134 @@
 import userData from "./UserData";
-import { Link } from "react-router-dom";
-import { Row, Button } from "react-bootstrap";
+import { useState } from "react";
+import { Row, Button, Modal } from "react-bootstrap";
 import UserCard from "../layout/UserCard";
+import { useForm } from "react-hook-form";
+import { supabase } from "../../utils/SupaBaseUtils";
+import { FetchingButton } from "../layout/Buttons/FetchingButton";
+import { CountryDropdown, RegionDropdown } from "react-country-region-selector";
 import "./Stories.css";
 export default function Stories() {
+  const [show, setShow] = useState(false);
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
+  const [country, setCountry] = useState("");
+  const [region, setRegion] = useState("");
+  const {
+    register,
+    reset,
+    handleSubmit,
+    formState: { errors },
+  } = useForm({
+    mode: "onChange",
+    defaultValues: {
+      title: "",
+      desc: "",
+    },
+  });
+
+  const onSubmit = async (fields) => {
+    let loading = true;
+    const { data, error } = await supabase.from("stories").insert({
```

Comment:
if you want you could use the wrapper from this library. I've been using it

[react-supabase](https://react-supabase.vercel.app/documentation/data/use-insert)